### PR TITLE
Upgraded prisoner transport bus

### PR DESCRIPTION
**Pull request by faefux**
#### Summary
Content "Upgraded Prisoner Transport Bus"

#### Purpose of change

Number 2 in adding and upgrading some  vehicles. I decided the prison bus didn't look very much like even an aproximation of a meat weagon, instead seemingly just a regular bus with contraband on.

#### Describe the solution

- Added a separate sealed compartment for driver and guards with extra seats, including a reclining chair for sleeping.
- Added a firearms lock-box in the guards compartment that spawns guns and ammo. (need help with this)
- Gave the prisoner compartment it's own entrance.
- Changed seating to the metal benches with shackles I added for pull #51645 (prisoner transport van)
- Removed contraband as it seems very unlikely you are sneaking contraband of any nature on to a sweat box (let alone some of the things in the list.) without plugging it, so unless everyone had chronic diarrhea the last time the vehicle was used..

#### Describe alternatives you've considered

Remaking it from ground up, making it wider to be more faithful to the images I have seen. 
Using a different design as there are many, I have no idea which is more prevalent IRL.

#### Testing

Tested on old install. Looks fine.

#### Additional context
I would have liked to add more guns in the box but when trying to add specifically `, "count": [ 1, 3 ]` to the below line this is incorrect.  Could someone advice what I should be using here instead?
` "items": [ { "x": 0, "y": 1, "chance": 70, "magazine": 100, "ammo": 50, "item_groups": [ "guns_cop" ] , "count": [ 1, 3 ] } ]`

Images:
image
Above: _The kind of bus I based it on. But I did not want to add a new set of plastic chairs so instead used to steel._
image
Above: _A screenshot of the new version of the bus._

### CODE REVIEW

**Comment by faefux** [comment]
Set as Draft. It makes things to messy unless I wait until #51645  is merged, I think.

**Comment by faefux** [comment]
Feel like something has gone horribly wrong here. No idea what. 
Github is a giant pain in the ass that constantly throws a huge spanner in the works of uploading very simple and easy things.. turning putting a few lines of simple json up into a nightmare of file weirdness and breaking everything that works absolutely fine before upload. It shouldn't go this wrong every time. Reeeee---

File: data/json/vehicle_groups.json [hunk 1]
```diff
@@ -31,6 +31,10 @@
       [ "policesuv_k9", 5 ],
       [ "police_k9tpt", 1 ],
       [ "truck_swat", 200 ],
+<<<<<<< HEAD
```
Comment (Zireael07) [inline data/json/vehicle_groups.json]: Bad merge?

Comment (faefux) [reply data/json/vehicle_groups.json]: I have literally no idea..  This is wht I never upload larger projects, *sigh*. Any advice on fixing this so I dont have to upload as a whole new branch or disregard?

**Pull request closed without merging**